projects/core/src/site-context/facade/base-site.service.ts

```diff
@@ -23,21 +23,29 @@ export class BaseSiteService implements SiteContext<string> {
    */
   getActive(): Observable<string> {
     return this.store.pipe(
-      select(SiteContextSelectors.getActiveBaseSite),
+      select(SiteContextSelectors.getActiveBaseSiteUid),
```

Comment:
I'm fine with introducing the new selector. What about start deprecating `getActiveBaseSite` rather than dropping?